### PR TITLE
Create nonfatal suggestions complaining about unknown fields in anaco…

### PR DESCRIPTION
**Pull request by havocp**
…nda-project.yml

### CODE REVIEW

**Comment by goanpeca** [comment]
LGTM, but is it a problem, or a suggestion?

**Comment by havocp** [comment]
I don't want it to be a fatal problem because if we add fields in the future it shouldn't break old versions of anaconda-project unnecessarily.

File: anaconda_project/project.py [hunk 1]
```diff
@@ -83,6 +83,13 @@ def _make_problems_into_objects(problems):
     return new_problems
 
 
+def _unknown_field_suggestions(project_file, problems, yaml_dict, known_fields):
+    for key in yaml_dict.keys():
+        if key not in known_fields:
+            problems.append(ProjectProblem(text="%s: Unknown field name '%s'" % (project_file.filename, key),
```
Comment (goanpeca) [inline anaconda_project/project.py]: Sure, I agree, but I saw this and was not sure if it was aproblem or a suggestion

Comment (havocp) [inline anaconda_project/project.py]: ah ok. it's made a suggestion by the only_a_suggestion=True kwarg. Kind of confusing

**Pull request merged**